.cbmc-batch/source/utils.c

```diff
@@ -179,3 +179,7 @@ uint64_t uninterpreted_hasher(const void *a) {
     assert(a != NULL);
     return __CPROVER_uninterpreted_hasher(a);
 }
+
+bool predicate_fn(uint8_t value) {
+    return nondet_bool();
```

Comment:
Hmm, shouldn't this actually be an uninterpreted function (so that it returns the same result whenever the same `value` is passed in?